.github/workflows/publish-mainline.yml

```diff
@@ -34,6 +34,6 @@ jobs:
       - name: Publish
         run: |
           cd packages/flame
-          node ./scripts/check-changed.js && yarn build && npm publish dist --quiet || exit 0
+          node ./scripts/check-changed.js && yarn release || exit 0
```

Comment:
Not sure why we weren't using `yarn release` before, hopefully this won't break the workflow 😉